module/os/windows/debug.c

```diff
@@ -125,12 +125,14 @@ printBuffer(const char *fmt, ...)
 	va_list args;
 	va_start(args, fmt);
 	char buf[max_line_length];
-	_snprintf(buf, 18, "%p: ", PsGetCurrentThread());
+	int buf_used;
```

Comment:
I'm not a fan of hardcoded numbers `18` so its a good thing to remove them and do it properly like you have...